PYME/Acquire/ui/tile_panel.py

```diff
@@ -93,7 +93,7 @@ def _on_stop(self, *args, **kwargs):
         self.scope.tiler.on_stop.disconnect(self._on_stop)
         self.scope.tiler.progress.disconnect(self._update)
         
-        wx.CallAfter(wx.CallLater,1e3, self._launch_viewer)
```

Comment:
Why such a large delay? Do we need to make this customisable and/or heuristic?

Reply:
Yeah this isn't really a solution, but a long enough delay to avoid #386 on small overviews